Formula/makedepend.rb

```diff
@@ -20,23 +19,23 @@ class Makedepend < Formula
   depends_on "pkg-config" => :build
 
   resource "xproto" do
-    url "https://xorg.freedesktop.org/releases/individual/proto/xproto-7.0.28.tar.gz"
-    mirror "http://xorg.mirrors.pair.com/individual/proto/xproto-7.0.28.tar.gz"
-    sha256 "6cabc8ce3fa2b1a2427871167b62c24d5b08a58bd3e81ed7aaf08f2bf6dbcfed"
+    url "https://xorg.freedesktop.org/releases/individual/proto/xproto-7.0.31.tar.gz"
+    mirror "http://xorg.mirrors.pair.com/individual/proto/xproto-7.0.31.tar.gz"
+    sha256 "6d755eaae27b45c5cc75529a12855fed5de5969b367ed05003944cf901ed43c7"
   end
 
   resource "xorg-macros" do
-    url "https://xorg.freedesktop.org/releases/individual/util/util-macros-1.19.0.tar.bz2"
-    mirror "http://xorg.mirrors.pair.com/individual/util/util-macros-1.19.0.tar.bz2"
-    sha256 "2835b11829ee634e19fa56517b4cfc52ef39acea0cd82e15f68096e27cbed0ba"
+    url "https://xorg.freedesktop.org/releases/individual/util/util-macros-1.19.2.tar.bz2"
+    mirror "http://xorg.mirrors.pair.com/individual/util/util-macros-1.19.2.tar.bz2"
+    sha256 "d7e43376ad220411499a79735020f9d145fdc159284867e99467e0d771f3e712"
   end
 
   def install
     resource("xproto").stage do
       system "./configure", "--disable-dependency-tracking",
                             "--disable-silent-rules",
                             "--prefix=#{buildpath}/xproto"
-      system "make", "install"
+      system "make", "install", "-j1"
```

Comment:
```suggestion
      ENV.deparallelize { system "make", "install" }
```

And please add the link above this line to your bug report